packages/default-storage-expo-plugin/package.json

```
@@ -0,0 +1,54 @@
+{
+  "name": "@react-native-async-storage/expo-plugin",
```

```suggestion
  "name": "@react-native-async-storage/async-storage-expo-plugin",
```
for future, if we ever have another expo plugin